llvm/tools/llvm-cxxfilt/Opts.td

```diff
@@ -27,4 +28,5 @@ def : F<"_", "Alias for --strip-underscore">, Alias<strip_underscore>;
 def : F<"h", "Alias for --help">, Alias<help>;
 def : F<"n", "Alias for --no-strip-underscore">, Alias<no_strip_underscore>;
 def : F<"p", "Alias for --no-params">, Alias<no_params>;
+def : F<"q", "Alias for --quote">, Alias<quote>;
```

Comment:
Please don't add single-character aliases for options that don't exist in the GNU version of the tool (`c++filt`), as it becomes a problem if they add their own -q option (we aim for compatibility with the GNU binutils). That is unless you are proposing to the binutils list to add it too...